src/main/java/com/gregtechceu/gtceu/integration/jade/provider/RecipeOutputProvider.java

```diff
@@ -144,31 +172,54 @@ protected void addTooltip(CompoundTag capData, ITooltip tooltip, Player player,
         }
     }
 
-    private void addItemTooltips(ITooltip iTooltip, List<ItemStack> outputItems) {
+    private void addItemTooltips(ITooltip iTooltip, List<Ingredient> outputItems) {
         IElementHelper helper = iTooltip.getElementHelper();
-        for (ItemStack itemOutput : outputItems) {
+        for (Ingredient itemOutput : outputItems) {
             if (itemOutput != null && !itemOutput.isEmpty()) {
-                int count = itemOutput.getCount();
-                itemOutput.setCount(1);
-                iTooltip.add(helper.smallItem(itemOutput));
-                Component text = Component.literal(" ")
-                        .append(String.valueOf(count))
+                ItemStack item = itemOutput.getItems()[0];
+                int count = item.getCount();
+                item.setCount(1);
+                iTooltip.add(helper.smallItem(item));
+                Component text;
```

Comment:
```suggestion
                MutableComponent text = CommonComponents.space();
```
this way you don't need to get the translated value on line 195, and can just do `text.append.....` instead